templates/authors/index.html

```diff
@@ -39,7 +39,7 @@
                 </div>
                 <div class='floatleft'>
                     <h2 class="inline">
-                        <a href="http://127.0.0.1:8000/site/authors/{{ target.url|urlencode:"/" }}">{{ target.displayName }}</a>
+                        <a href="{{ target.url|urlencode:"/" }}">{{ target.displayName }}</a>
```

Comment:
This should be "{{host}}/site/authors/{{target.url|urlencode:"/"}}" We need to be calling host/site so we redirect to the html page, not target.url by itself which would give us raw json